CONTRIBUTING.md

```diff
@@ -13,16 +13,23 @@ improvement.
 When you create your PR, please tag _Craig Baumgarten_ or 
```

Comment:
@MessagingMoore Craig's not the best choice here since he won't be involved, so who should replace him?

Reply:
@polarweasel @MessagingMoore I can leave it as just Serhii until later notice if that makes more sense